tools/ci_build/github/azure-pipelines/win-gpu-tensorrt-ci-pipeline.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       displayName: 'Download test data and generate cmake config'
```

Comment:
>Download test data and generate cmake config' [](start = 20, length = 45)

Which part does the download? On my dev box I usualy have to do it manually

Reply:
No, It didn't download the test data.

Reply:
Should the name of the task be changed then? It does mention Download test data

---
In reply to: [337250310](https://github.com/microsoft/onnxruntime/pull/2211#discussion_r337250310) [](ancestors = 337250310)

Reply:
I'll submit a PR to fix it.